internal/credentials/credentials.go

```
@@ -0,0 +1,341 @@
+		return err
+	}
+	if err := os.MkdirAll(filepath.Dir(path), 0700); err != nil {
+		return err
+	}
+	body := fmt.Sprintf("[default]\nprofile = %s\n", name)
+	tmp, err := os.CreateTemp(filepath.Dir(path), ".config-*.tmp")
+	if err != nil {
+		return err
+	}
+	if _, err := tmp.Write([]byte(body)); err != nil {
+		tmp.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `tmp.Close` is not checked

(errcheck)